pkg/management/component.go

```diff
@@ -1795,6 +1801,8 @@ func (c *component) UpdateAuthorizations(ctx context.Context, realmName string,
 		}
 		c.logger.Info(ctx, "msg", "Updated authorizations", "add_count", len(addAuthz), "del_count", len(delAuthz))
 
+		c.kafkaProducer.SendMessageBytes([]byte{})
+
```

Comment:
Don't you want to update authorization in the authorization manager in the same time? Or add a Kafka consumer to check for new authorizations?

Reply:
yes, the plan is to add a kafka consumer to keycloak bridge too. For now I only implemented it in the accreditations-service